.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
```

Comment:
@jgedarovich found this via googling. it seems to have calm the checks. you'll want to verify the legit-ness of it.